documentation/contributing/future_projects/index.markdown

```diff
@@ -15,62 +15,101 @@ This page lists potential projects that would greatly benefit the MoveIt project
 
-Once a standard benchmark has been established, subsequent work can focus on running the benchmark suite as part of a continuous integration suite. Additional work can be done on tuning hyperparameter settings that will lead to better overall performance.
+  Once a standard benchmark has been established, subsequent work can focus on running the benchmark suite as part of a continuous integration suite. Additional work can be done on tuning hyperparameter settings that will lead to better overall performance.
 
 ## MoveIt 2 Tutorials
 
-One of the reasons MoveIt is so popular is that the tutorials make it very easy to get started. MoveIt 2 beta will be released in February and the tutorials needs to be updated for this new version.
+- List of prerequisites: ROS, some motion planning background
+- Description of programming skills: C++
+- Difficulty level: Medium
+- List of potential mentors: Mike Lautman, Mark Moll
+- Detailed description: One of the reasons MoveIt is so popular is that the tutorials make it very easy to get started. MoveIt 2 beta will be released in February and the tutorials needs to be updated for this new version.
 
 ## Add Support for MoveIt Task Constructor to MoveIt 2
 
-The [MoveIt Task Constructor](https://ros-planning.github.io/moveit_tutorials/doc/moveit_task_constructor/moveit_task_constructor_tutorial.html) framework provides a flexible and transparent way to define and plan actions that consist of multiple interdependent subtasks. It draws on the planning capabilities of MoveIt to solve individual subproblems in black-box planning stages. Porting this MoveIt 2 may also include porting MoveIt 1's MoveGroup interface to MoveIt 2.
+- List of prerequisites: ROS, some motion planning background
+- Description of programming skills: C++
+- Difficulty level: Medium
+- List of potential mentors: Robert Haschke, Henning Kayser
+- Detailed description: The [MoveIt Task Constructor](https://ros-planning.github.io/moveit_tutorials/doc/moveit_task_constructor/moveit_task_constructor_tutorial.html) framework provides a flexible and transparent way to define and plan actions that consist of multiple interdependent subtasks. It draws on the planning capabilities of MoveIt to solve individual subproblems in black-box planning stages. Porting this MoveIt 2 may also include porting MoveIt 1's MoveGroup interface to MoveIt 2.
 
 ## Improved Integration with ROS-Controls and Controller Switching
 
-Utilize low-level controller switching (position/velocity/force-torque) during execution of MoveIt plans. Improve the [ROSControlInterface plugin](https://github.com/ros-planning/moveit/tree/melodic-devel/moveit_plugins/moveit_ros_control_interface). Documentation on how to combine the components.
+- List of prerequisites: ROS, some background in control theory
+- Description of programming skills: C++
+- Difficulty level: Medium
+- List of potential mentors: Andy Zelenak 
```

Comment:
```suggestion
- List of potential mentors: Andy Zelenak, Robert Haschke 
```
My students are already working on controller switching in the context of MTC. See https://github.com/ros-planning/moveit/pull/1832.